README.md

```diff
@@ -1,28 +1,87 @@
-# tf-aws-iam-cloudability
+# Configure account for cloudability monitoring
 
-Terraform module that creates the Cloudability Role for a Linked Account
+## Prerequisites
 
-## Requirements
+* Get External ID from cloudability in the form of a uuid. This UUID can be found in the cloudformation stack at the cloudability console.
 
- * Cloudability `external id` assigned to your account
+## Configure
 
-## Usage
+Add a cloudability.tf to the account directory
+
+```
+variable "cloudability_external_id" {
+  description = "Cloudability External id, provided by cloudability"
+  default     = ""
+}
 
-```hcl
 module "cloudability" {
-  source      = "git::https://github.com/edrans/tf-aws-iam-cloudability.git?ref=master"
-  external_id = "99999999-8888-7777-6666-555555555555"
+  source      = "../modules/tf-aws-cloudability"
+  external_id = "${var.cloudability_external_id}"
 }
 ```
 
+Add the External ID to  the tfvars file (config.auto.tfvars)
+```
+cloudability_external_id = "6124f6fd-6964-4fa6-9047-xxxxxxxxxxx"
+```
+
+_Without the external_id the role is not created_
+
+## Role already exists
+
+When it already exists terraform gives this error:
+```
+* aws_iam_role.cloudability_role: Error creating IAM Role CloudabilityRole: EntityAlreadyExists: Role with name CloudabilityRole already exists.
+	status code: 409, request id: xxxxxx-xxxx-xxxx-xxxx-xxxxxxxxx
+```
+
+### Import The role:
+
+To import the resource Terraform has the following command available:
+`terraform  import  module.cloudability.aws_iam_role.cloudability_role  CloudabilityRole`
+
+The wrapper script has this option to include the assume role and run import for this account
+`./runall.nl -b <accountid-description> -t module.cloudability.aws_iam_role.cloudability_role  -v CloudabilityRole`
```

Comment:
Where this command comes from?

Reply:
It seems, this code is not necessary, It has heritance from other repo, done!

Reply:
Apparently that command is not necessary anymore